.eslintrc.js

```diff
@@ -54,7 +54,7 @@ module.exports = {
     'deprecation/deprecation': 'error',
 
     // TODO(#7138): Investigate enabling these soon ✨
-    '@typescript-eslint/consistent-indexed-object-style': 'off',
+    '@typescript-eslint/consistent-indexed-object-style': 'error',
```

Comment:
No need to leave this here under a TODO 😄 it's enabled as part of `stylistic-type-checked`.

```suggestion
```